src/tests/ftest/launch.py

```diff
@@ -1149,11 +1153,11 @@ def rename_logs(avocado_logs_dir, test_file):
         avocado_logs_dir (str): avocado job-results directory
         test_file (str): the test python file
     """
-    test_name = get_test_category(test_file)
     test_logs_lnk = os.path.join(avocado_logs_dir, "latest")
     test_logs_dir = os.path.realpath(test_logs_lnk)
-    new_test_logs_dir = "{}-{}".format(test_logs_dir, test_name)
+    new_test_logs_dir = os.path.join(avocado_logs_dir, test_file)
     try:
+        os.makedirs(new_test_logs_dir)
```

Comment:
I more than suspect this will be undesirable when developers are running manually.  Since a test like `aggregation/aggregation_basic.py` will now always be placed in `avocado/job-results/aggregation/aggregation_basic.py` multiple runs of the same test will overwrite previous runs.  With the default avocado directory (`test_logs_dir` in this code) we have a unique `job-<date_time_hash>` directory for each run.

This is another case for a CI-specific `launch.py` mode driven by a command line option.

Reply:
yep, will try to use cmdline option for this. thanks.